src/common/test/data/journalArticle-single.html

```diff
@@ -0,0 +1,53 @@
+	<title>Scarcity or Abundance? Preserving the Past in a Digital Era</title>
+</head>
+<body>
+<div style="line-height: 1.35; padding-left: 2em; text-indent:-2em;" class="csl-bib-body">
+	<div class="csl-entry">Rosenzweig, Roy. 2003. “Scarcity or Abundance? Preserving the Past in a Digital Era.” <i>The American Historical Review</i> 108 (3): 735–62. doi:10.2307/3523084.</div>
+	<span class="Z3988" title="url_ver=Z39.88-2004&amp;ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fzotero.org%3A2&amp;rft_id=info%3Adoi%2F10.2307%2F3523084&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.atitle=Scarcity%20or%20Abundance%3F%20Preserving%20the%20Past%20in%20a%20Digital%20Era&amp;rft.jtitle=The%20American%20Historical%20Review&amp;rft.volume=108&amp;rft.issue=3&amp;rft.aufirst=Roy&amp;rft.aulast=Rosenzweig&amp;rft.au=Roy%20Rosenzweig&amp;rft.date=2003-06&amp;rft.pages=735-762&amp;rft.spage=735&amp;rft.epage=762&amp;rft.issn=00028762"></span>
+</div></body>
+<!-- Can't inject into chrome-extension:// pages -->
+<script src='/node_modules.js'></script>
+<script src='/zotero.js'></script>
+<script src='/zotero_config.js'></script>
+<script src='/promise.js'></script>
+<script src='/http.js'></script>
+<script src='/proxy.js'></script>
+<script src='/cachedTypes.js'></script>
+<script src='/zotero/date.js'></script>
+<script src='/zotero/debug.js'></script>
+<script src='/zotero/openurl.js'></script>
+<script src='/zotero/xregexp/xregexp.js'></script>
+<script src='/zotero/xregexp/addons/build.js'></script>
+<script src='/zotero/xregexp/addons/matchrecursive.js'></script>
+<script src='/zotero/xregexp/addons/unicode/unicode-base.js'></script>
+<script src='/zotero/xregexp/addons/unicode/unicode-categories.js'></script>
+<script src='/zotero/xregexp/addons/unicode/unicode-zotero.js'></script>
+<script src='/zotero/rdf/init.js'></script>
+<script src='/zotero/rdf/uri.js'></script>
+<script src='/zotero/rdf/term.js'></script>
+<script src='/zotero/rdf/identity.js'></script>
+<script src='/zotero/rdf/match.js'></script>
+<script src='/zotero/rdf/rdfparser.js'></script>
+<script src='/zotero/translation/translate.js'></script>
+<script src='/zotero/translation/translator.js'></script>
+<script src='/translate_item.js'></script>
+<script src='/zotero/connectorTypeSchemaData.js'></script>
+<script src='/zotero/utilities.js'></script>
+<script src='/zotero/utilities_translate.js'></script>
+<script src='/inject/http.js'></script>
+<script src='/inject/progressWindow.js'></script>
+<script src='/inject/translate_inject.js'></script>
+<script src='/messages.js'></script>
+<script src='/messaging_inject.js'></script>
+<script src='/api.js'></script>
+<script src='/inject/inject.js'></script>
+<script src='/test/testSetup.js'></script>
+<script src='/lib/sinon.js'></script>
+<script src='/lib/react.js'></script>
+<script src='/lib/react-dom.js'></script>
+<script src='/ui/modal-prompt.js'></script>
```

Comment:
https://github.com/zotero/zotero-connectors/pull/143/files/57024974d0e5fc4e9b8ae657a8246724f3fda683..3665c6e54232ae4d183beb51ecc19a3059d21d32#diff-9e0e8bc9053bf2e3214ad5175f823c6eR12

We could maybe do this in the build step, but some of this is "programatically injected" during normal operation. I don't have strong feelings.